src/ui/Analytics/AnalyticsInformation.ts

```diff
@@ -0,0 +1,37 @@
+import { history } from 'coveo.analytics';
+import { findLastIndex } from 'underscore';
+import { Cookie } from '../../utils/CookieUtils';
+
+export class AnalyticsInformation {
```

Comment:
Hopefully this is what you were thinking about, @samisayegh ?

This works the same regardless of where it's instantiated. Privacy is now managed by not initializing cookies and clearing them if they already exist.

Reply:
Thanks Benjamin,